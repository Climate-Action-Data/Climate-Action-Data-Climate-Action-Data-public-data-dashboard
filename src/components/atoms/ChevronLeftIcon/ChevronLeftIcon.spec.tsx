import { render } from '@testing-library/react'
import { ChevronLeftIcon } from './ChevronLeftIcon'

it(`renders correctly`, () => {
  const { container } = render(<ChevronLeftIcon />)
  expect(container).toMatchSnapshot()
})
