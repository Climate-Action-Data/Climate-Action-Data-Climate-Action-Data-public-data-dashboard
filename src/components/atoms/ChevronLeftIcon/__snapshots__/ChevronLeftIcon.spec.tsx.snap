// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <svg
    class="chakra-icon css-n059si"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M13.7402 17.1274L9.24601 12.6332C9.15241 12.5396 9.08638 12.4392 9.04793 12.3322C9.00947 12.2252 8.99023 12.1146 8.99023 12.0005C8.99023 11.88 9.00947 11.7678 9.04793 11.664C9.08638 11.5601 9.15241 11.4614 9.24601 11.3678L13.7402 6.87358C13.8851 6.72869 14.0607 6.65625 14.2671 6.65625C14.4735 6.65625 14.6492 6.72869 14.794 6.87358C14.9389 7.01844 15.0114 7.19408 15.0114 7.40048C15.0114 7.60688 14.9389 7.78251 14.794 7.92738L10.721 12.0005L14.794 16.0736C14.9389 16.2184 15.0114 16.3941 15.0114 16.6005C15.0114 16.8069 14.9389 16.9825 14.794 17.1274C14.6492 17.2723 14.4735 17.3447 14.2671 17.3447C14.0607 17.3447 13.8851 17.2723 13.7402 17.1274Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
